lib/service_check.pm

```diff
@@ -54,11 +60,12 @@ our $default_services = {
         support_ver   => '12-SP3,12-SP4,15,15-SP1'
     },
     # Quick hack for poo 50576, we need this workround before full solution
-    #    apache => {
-    #    srv_pkg_name  => 'apache2',
-    #    srv_proc_name => 'apache2',
-    #    support_ver   => '12-SP3,12-SP4,15,15-SP1'
-    #},
+    apache => {
+        srv_pkg_name       => 'apache2',
+        srv_proc_name      => 'apache2',
+        support_ver        => '12-SP3,12-SP4,15,15-SP1',
+        service_check_func => 'full_apache_check'
```

Comment:
why not using a function-pointer instead of using the function name?
```suggestion
        service_check_func => \&full_apache_check
```
The call later  on would be more readable:
```
$service->{$s}->{service_check_func}->('before');
```